wikiconv/conversation_reconstruction/dataflow_test.py

```diff
@@ -15,80 +16,143 @@
 import dataflow_main
 
 
+class FakeStorageClient(object):
+  pass
+
+
 class DataflowTest(unittest.TestCase):
 
   def test_index_by_page_id(self):
     data = '{"page_id": "deadbeef", "content": "Abracadabra"}'
     actual = dataflow_main.index_by_page_id(data)
-    self.assertTupleEqual(('deadbeef', {
-        'page_id': 'deadbeef',
-        'content': 'Abracadabra'
+    self.assertTupleEqual(("deadbeef", {
+        "page_id": "deadbeef",
+        "content": "Abracadabra"
     }), actual)
 
   def test_index_by_rev_id(self):
     data = '{"rev_id": "deadbeef", "content": "Abracadabra"}'
     actual = dataflow_main.index_by_rev_id(data)
-    self.assertTupleEqual(('deadbeef', data), actual)
+    self.assertTupleEqual(("deadbeef", data), actual)
 
   def test_mark_revisions_of_big_pages(self):
```

Comment:
Do we still need to mark revisions of big pages if we use BigTable?

Reply:
Not sure yet, haven't dug into this deeply yet. Wanted test coverage to be sure I don't break anything. - I know big pages may not be well covered by the unit tests.

I'm thinking of lowering the big pages size to something tiny, like 200 bytes, to improve the testing. Right now the code path that use the storage mock isn't touched.

That will be a future change.